pkg/interfaces/contracts/liquidity-mining/ILiquidityGauge.sol

```diff
@@ -28,4 +32,12 @@ interface ILiquidityGauge {
     function killGauge() external;
 
     function unkillGauge() external;
+
+    function set_max_relative_weight(uint256 maxRelativeWeight) external;
+
+    function get_capped_relative_weight(uint256 time) external view returns (uint256);
+
+    function get_current_capped_relative_weight() external view returns (uint256);
+
+    function get_absolute_max_relative_weight() external pure returns (uint256);
```

Comment:
I'd consider getting rid of this tbh, we use 18 decimals everywhere

Reply:
yes, but given that the `set` method reverts if you pass a value above this one, shouldn't we keep it JIC?